nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -162,7 +162,8 @@ define([
         }
 
         let kbaseNode = null, // the DOM element used as the container for everything in this cell
-            cellBus = null,
+            cellBus = null, // the parent cell bus that gets external messages
+            controllerBus = null, // the main bus for this cell and its children
```

Comment:
The cellBus is intended to talk to the rest of the page, and deal with events that affect the actual encapsulating cell, like deletions. The controllerBus is intended to communicate this modules children, like the configure tab, which will send up a message saying the configuration's complete or not.